assets/pages/new-contributions-landing/contributionsLandingReducer.js

```diff
@@ -86,9 +86,9 @@ export type State = {
 
 function createFormReducer(countryGroupId: CountryGroupId) {
   const amountsForCountry: { [Contrib]: Amount[] } = {
-    ONE_OFF: amounts('notintest').ONE_OFF[countryGroupId],
-    MONTHLY: amounts('notintest').MONTHLY[countryGroupId],
-    ANNUAL: amounts('notintest').ANNUAL[countryGroupId],
+    ONE_OFF: amounts('control').ONE_OFF[countryGroupId],
```

Comment:
initialise here with the control - this will be updated by `ContributionAmount.jsx` when the variant name is available

Reply:
why did you need to do this?

Reply:
the form needs to be initialised with something. I've just changed the name of the variant used for the initialisation

Reply:
I could make the parameter here optional (defaulting to `control`), so that we don't have to pass it in at this point

Reply:
what was wrong with notintest?

Reply:
ah, I removed it from the list of valid values so it had to be replaced here

Reply:
hmmmm. I'm pretty sure that the abtest framework can still assign people notintest though, theoretically no? Why did you have to remove it in this scenario?

Reply:
ok, I didn't realise `notintest` is a special value. All users will now see the annual contribution option, so I thought it made sense to default to `control`. I can add `notintest` back in if it's better? (and default to the same settings as the `control`)

Reply:
Yes, our usual pattern is to name the variants in the test as control + whatever your variant names are, then change the default behaviour for the variants. I.e you shouldn't really be doing something like `if (variantName === 'control') {...` (which I don't think you are doing). But you should be aware that if the ab testing framework fails to assign someone (usually due to a location constraint or something) then it assigns them as 'notintest'. So it's probably best  to put it in there, even though in this case everyone will be in one test or the other

Reply:
ok, thanks!